### ISSUE TITLE
Compilation fails if directory isn't called 'vlab'

### ISSUE THREAD

**Issue by gilgamec** [AlgorithmicBotany/vlab#2]
If the base directory this repository is checked out to isn't called `vlab`, then compilation fails because nothing in `libs/*` is in the include path. (For me, the first error is that `delete_recursive.h` isn't found when compiling `gallery`.)  I think this happens because the directory name is hardcoded in the `.pro` files (`MY_BASE = ../../vlab`), though I may be wrong.

**Comment by mikcieslak** [comment]
Yes, you are right. For now, it is required to use the root directory `vlab`. I'll leave the issue open until it can be addressed.
